recipes/codemodel/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "codemodel" %}
+{% set version = "0.0.5" %}
+{% set sha256 = "d3d9714454981542beb75fad3d601365bfdf075b93a7530fd399f1c3e06dce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - astor
+    - black
+    - isort
+
+test:
+  imports:
+    - codemodel
+    - codemodel.tests
```

Comment:
Only test the main module here, tests don't need to be import-tested and most often shouldn't either as they have additional requirements that are not necessary for running the main code.
```suggestion
```